modules/virt-example-node-placement-tolerations-hyperconverged-cr.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/install/virt-specifying-nodes-for-virtualization-components.adoc
+
+[id="virt-example-node-placement-tolerations-hyperconverged-cr_{context}"]
+= Example: Node placement with tolerations in the HyperConverged Cluster CR
+
+In this example, nodes that are intended for virtualization resources are labeled with the `key=virtualization:NoSchedule` taint. Only pods with the matching tolerations are scheduled to these nodes.
+
+[source,yaml]
+----
+...
+spec:
+  workloads:
+    nodePlacement:
+      tolerations:
+      - key: “key”
```

Comment:
can we use the simple ascii quote for copy-pastability?

Reply:
Wow, thanks for pointing this out -- I didn't realize they were fancy quotes!